From 6bf17fd6477effe633ebaa31f68fc52bb037b42e Mon Sep 17 00:00:00 2001
From: Seth Jennings <sjenning@redhat.com>
Date: Mon, 18 Aug 2014 15:42:43 -0400
Subject: [edac] sb_edac: Degrade log level for device registration

Message-id: <1408376571-8244-5-git-send-email-sjenning@redhat.com>
Patchwork-id: 87859
O-Subject: [RHEL7.1 PATCH 04/12] sb_edac: Degrade log level for device registration
Bugzilla: 1127019
RH-Acked-by: Aristeu Rozanski <aris@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiang Liu <jiang.liu@linux.intel.com>

Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7802951
Bugzilla: https://brewweb.devel.redhat.com/taskinfo?taskID=7802951
Upstream: ec5a0b38247a228f69d1d12c8ca0d6fe54b42827

On a system with four Intel processors, it generates too many messages
"EDAC sbridge: Seeking for: dev 1d.3 PCI ID xxxx". And it doesn't give
many useful information for normal users, so change log level from INFO
to DEBUG.

Signed-off-by: Jiang Liu <jiang.liu@linux.intel.com>
Link: http://lkml.kernel.org/r/1392613824-11230-1-git-send-email-jiang.liu@linux.intel.com
Acked-by: Aristeu Rozanski <aris@redhat.com>
Signed-off-by: Borislav Petkov <bp@suse.de>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/edac/sb_edac.c b/drivers/edac/sb_edac.c
index d9a6214..f28822f 100644
--- a/drivers/edac/sb_edac.c
+++ b/drivers/edac/sb_edac.c
@@ -1283,7 +1283,7 @@ static int sbridge_get_onedevice(struct pci_dev **prev,
  struct pci_dev *pdev = NULL;
  u8 bus = 0;
 
- sbridge_printk(KERN_INFO,
+ sbridge_printk(KERN_DEBUG,
   "Seeking for: dev %02x.%d PCI ID %04x:%04x\n",
   dev_descr->dev, dev_descr->func,
   PCI_VENDOR_ID_INTEL, dev_descr->dev_id);
-- 
1.7.1